tor/core/user_interaction.py

```diff
@@ -88,7 +88,10 @@ def process_claim(post, config):
 
         if flair.unclaimed in top_parent.link_flair_text:
             # need to get that "Summoned - Unclaimed" in there too
-            post.reply(_(claim_success))
+            if first_claim:
+                post.reply(_(claim_success.format(first_claim_success)))
+            else:
+                post.reply(_(claim_success))
```

Comment:
We'll want to format this with an empty string, otherwise the `{}` will show up in the final product. Example:

```python
>>> x = "Test!{}Test!"
>>> print(x)
Test!{}Test!
>>> print(x.format(''))
Test!Test!
>>>
```

Reply:
Ah good point :+1: